pkg/controller/installation/products/rhsso/reconciler.go

```diff
@@ -197,7 +183,13 @@ func (r *Reconciler) handleProgressPhase(ctx context.Context, serverClient pkgcl
 			return v1alpha1.PhaseFailed, errors.Wrap(err, "failed to write rhsso config")
 		}
 
-		r.logger.Info("keycloak has successfully processed the keycloakRealm")
+		err = r.setupOpenshiftIDP(ctx, kcr, serverClient)
+		if err != nil {
+			logrus.Errorf("Failed to setup Openshift IDP %v", err)
```

Comment:
rather than logging here, which can create extra noise I would prefer we wrap the error and return it up the call stack